docs/java/java-tutorial.md

```diff
@@ -22,7 +22,7 @@ If you need to install VS Code, [download it here](https://code.visualstudio.com
 
 ## Supported Java versions
 
-VS Code works with all major versions of Java SE from 11 and supports projects with version 1.5 and above. For more details, refer to [Configure JDK](/docs/java/java-project.md#configure-jdk).
+Supported versions for running VS Code for Java and supported versions for your projects are two seperate concepts. To run VS Code for Java, Java SE 11 or above version is required; for projects, VS Code for Java supports projects with version 1.5 or above. For more details, refer to [Configure JDK](/docs/java/java-project.md#configure-jdk).
```

Comment:
@yaojinyang can you please clarify what exactly you mean by "VS Code for Java" here? Is it a specific extension, the extension pack, or perhaps the Java LSP? I was reading this page and got a little confused by this passage.